examples/platforms/nrf528xx/nrf52811/openthread-core-nrf52811-config.h

```diff
@@ -160,6 +160,16 @@
 #define OPENTHREAD_CONFIG_MAC_SOFTWARE_TX_TIMING_ENABLE (OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2)
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_MAC_SOFTWARE_RX_TIMING_ENABLE
+ *
+ * Define to 1 to enable software reception target time logic.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_MAC_SOFTWARE_RX_TIMING_ENABLE
+#define OPENTHREAD_CONFIG_MAC_SOFTWARE_RX_TIMING_ENABLE (OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2)
```

Comment:
Is it safe to use `OPENTHREAD_CONFIG_THREAD_VERSION` in the project config? The defaults are included after the project config.

https://github.com/openthread/openthread/blob/8467a3d5223c274b65a71080f6aad94dc182a9bb/src/core/openthread-core-config.h#L45-L51

Reply:
It seems not. This was just inheriting convention from previous TX capabilities.

I would say the right way should be to just confirm or deny the capabilities in the platforms configs (`0` or `1`) and make sure they are used correctly within OT code depending on the configured Thread version.

But could we address this in a different PR?

Reply:
I am OK to resolve in a separate PR.